lua/lazyvim/plugins/extras/lang/vue.lua

```diff
@@ -38,12 +38,16 @@ return {
           },
         },
       })
-
-      LazyVim.extend(opts.servers.vtsls, "settings.vtsls.tsserver.globalPlugins", {
-        name = "@vue/typescript-plugin",
-        location = vue_typescript_plugin,
-        languages = { "vue" },
+      LazyVim.extend(opts.servers.vtsls.settings.vtsls.tsserver.globalPlugins, {
+        {
```

Comment:
This should be one value, not a list of values, so you shoudl remove one set of `{}`

Reply:
Seems like I actually did it wrong, so that was my mistake